src/main/kotlin/no/nav/syfo/client/norg/NorgClient.kt

```diff
@@ -62,12 +62,11 @@ class NorgClient(
             COUNT_CALL_NORG_ENHET_SUCCESS.increment()
             return response
         } catch (e: ResponseException) {
-            val message = "Call to NORG2 for overordnet enhet failed with status HTTP-${e.response.status} for enhet $enhet, callId=$callId"
             if (e.response.status == HttpStatusCode.NotFound) {
-                log.warn(message)
                 COUNT_CALL_NORG_ENHET_NOT_FOUND.increment()
```

Comment:
Vil vi ikke logge not found lenger?

Reply:
Tenker at dette er normal siden en del veiledere/Modia-brukere tydeligvis hører til fylkeskontor.